src/main/java/oss/fosslight/service/ApiVulnerabilityService.java

```diff
@@ -9,7 +9,7 @@
 import java.util.Map;
 
 public interface ApiVulnerabilityService {	
-	public List<Map<String, Object>> selectNvdList(String product, String version);
+	public List<Map<String, Object>> selectNvdList(String product, String version, String Id);
```

Comment:
@MiniVee , Parameter must begin with a lower-case letter.

Reply:
@jongwooo OMG....😅 I got a mistake!! I'll fix it. thank you!!!